plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/GeoDistanceTests.java

```diff
@@ -222,12 +232,15 @@ public void testSimpleDistance() throws Exception {
     }
 
     public void testDistanceSortingMVFields() throws Exception {
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_1_0_0, Version.CURRENT);
```

Comment:
I like that idea. I renamed issue #14562. That issue will cover removing all of these random versions and adding dedicated backcompat testing. For now I'll change the random version range as follows:

* for master: Version.V_2_0_0 to Version.CURRENT
* for 2.x: Version.V_1_0_0 to Version.CURRENT

This way we cover backcompat testing with supported versions and increase coverage for the new code in master.